libappstream-builder/asb-context.c

```diff
@@ -565,12 +565,14 @@ asb_context_setup (AsbContext *ctx, GError **error)
 	/* icons is nuked; we can re-decompress from the -icons.tar.gz */
 	if (!asb_utils_ensure_exists (priv->icons_dir, error))
 		return FALSE;
+
+	g_autofree gchar *icons_dir_lodpi = NULL;
```

Comment:
this needs to be at the start of the block like the others

Reply:
Indeed